Taskfile.yml

```diff
@@ -7,6 +7,18 @@ tasks:
       - task: clean-agents
       - task: clean-server
 
+  lint:
+    desc: lint-agent
+    env:
+      PYTHONPATH: "./agents/ten_packages/system/ten_runtime_python/lib:./agents/ten_packages/system/ten_runtime_python/interface:./agents/ten_packages/system/ten_ai_base/interface"
+    cmds:
+      - ./agents/scripts/pylint.sh
+
+  install-tools:
+    desc: install tools
+    cmds:
+      - pip install pylint
```

Comment:
Put it in `ten_agent_build` seems better. Next one will have it. https://github.com/TEN-framework/ten_agent_build/pull/14